tox.ini

```diff
@@ -1,8 +1,28 @@
 [tox]
-envlist = py26,py27,py33,py34,py35,pypy
+envlist = py26,py27,py33,py34,py35,pypy,lint
 
 [testenv]
 deps=
-	pytest	
+    pytest
+    PyYAML
```

Comment:
this is very dangerous - as it was one of the reasons we forked from bunch in the first place. It had a bug where it needed YAML and would not work without it installed. YAML is a very heavy dependency and should not be required by munch. Having it in the test deps means we will never spot hidden reliance on it. At the very least have multiple configurations in which some install YAML and some don't.